tests/unit/helpers.js

```diff
@@ -1741,3 +1746,49 @@ export const renderPage = (options = {}) => {
 
 export const getSearchErrorHandlerId = (page) =>
   `src/amo/components/Search/index.js-${page}`;
+
+export const fakeAuthors = [
+  { ...fakeAuthor, name: 'Bob', username: 'bsilverberg', id: 1 },
+  { ...fakeAuthor, name: 'Matt', username: 'tofumatt', id: 2 },
+];
+
+export const loadAddonsByAuthors = ({
+  addonName = 'Some add-on name',
+  addonType = ADDON_TYPE_EXTENSION,
+  count = null,
+  forAddonSlug = 'some-slug',
+  multipleAuthors = false,
+  store,
+}) => {
+  const pageSize =
+    addonType === ADDON_TYPE_STATIC_THEME
+      ? THEMES_BY_AUTHORS_PAGE_SIZE
+      : EXTENSIONS_BY_AUTHORS_PAGE_SIZE;
+
+  const addons = [];
+  const totalAddons = typeof count === 'number' ? count : pageSize;
```

Comment:
for my interest, why `typeof` rather than testing that `count === undefined` or whatever?

Reply:
I assume it's to easily cover all cases where it isn't a number (e.g., `null`, `undefined`, a string, etc.)